syne_tune/callbacks/tensorboard_callback.py

```diff
@@ -106,11 +110,12 @@ def on_trial_result(
             if metric not in self.ignore_metrics:
                 self.writer.add_scalar(metric, result[metric], self.iter, walltime)
 
-        for key, value in trial.config.items():
-            if isinstance(value, numbers.Number):
-                self.writer.add_scalar(key, value, self.iter, walltime)
-            else:
-                self.writer.add_text(key, str(value), self.iter, walltime)
+        if self.log_hyperparameters:
+            for key, value in trial.config.items():
+                if isinstance(value, numbers.Number):
+                    self.writer.add_scalar(key, value, self.iter, walltime)
+                else:
+                    self.writer.add_text(key, str(value), self.iter, walltime)
 
         self.writer.add_scalar("runtime", result[ST_TUNER_TIME], self.iter, walltime)
```

Comment:
To be safe, I'd check whether "runtime" does not clash with any HP name

Reply:
I'll send a new PR for this